pom.xml

```diff
@@ -137,6 +137,11 @@
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>jackson2-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.j2objc</groupId>
```

Comment:
Shouldn't this be in the `dependencyManagement` section?

Reply:
taken care of this review comment